src/HtmlQuotations.ts

```diff
@@ -112,33 +111,41 @@ export function deleteQuotationTags(document: Document, element: Node, quotation
     isTagInQuotation
   };
 }
+export interface cutQuoteOption {
```

Comment:
Types should be uppercase.
Also plural in this case.

```suggestion
export interface CutQuoteOptions {
```